nuxeo-jsf/nuxeo-platform-forms-layout-client/src/main/java/org/nuxeo/ecm/platform/forms/layout/facelets/plugins/AbstractDirectorySelectWidgetTypeHandler.java

```diff
@@ -86,4 +95,27 @@ protected boolean shouldAddWidgetPropsHandler(Widget widget) {
         return props;
     }
 
+    /**
+     * @since 9.1 Get tag attributes for a specific mode
+     * @param widget the widget to generate tag attributes for
+     * @param mode the given mode like PLAIN, CSV ...
+     * @param helper an instance of FaceletHandlerHelper
+     * @param widgetId the widget id
+     * @return list of tag attributes for the given widget
+     */
```

Comment:
Please follow javadoc conventions and start by a sentence finishing by a dot (here "Get tag attributes for a specific mode."). The since annotation should be on a distinct line, the return is not necessary if it's basically repeating what the first sentence said, and parameters do not need to be detailed if you do not have much to say that is not already clear enough given the parameters name and type.